src/cljdoc/render/layout.clj

```diff
@@ -84,27 +84,35 @@
     (into [:div.absolute.top-0.bottom-0.left-0.right-0.overflow-y-scroll.ph4-ns.ph2.main-scroll-view]
           content)])
 
-(defn top-bar [cache-id scm-url]
-  [:nav.pv2.ph3.pv3-ns.ph4-ns.bb.b--black-10.flex.items-center
-   [:a.dib.v-mid.link.dim.black.b.f6.mr3 {:href (routes/url-for :artifact/version :path-params cache-id)}
-    (util/clojars-id cache-id)]
-   [:a.dib.v-mid.link.dim.gray.f6.mr3
-    {:href (routes/url-for :artifact/index :path-params cache-id)}
-    (:version cache-id)]
-   [:a {:href "/"}
-    [:span.link.dib.v-mid.mr3.pv1.ph2.ba.hover-blue.br1.ttu.fw5.f7.silver.tracked "cljdoc Beta"]]
-   [:a.silver.link.hover-blue.ttu.fw5.f7.tracked.pv1
-    {:href (util/github-url :issues)}
-    "Have Feedback?"]
-   [:div.tr
-    {:style {:flex-grow 1}}
-    [:form.dib.mr3 {:action "/api/request-build2" :method "POST"}
-     [:input.pa2.mr2.br2.ba.outline-0.blue {:type "hidden" :id "project" :name "project" :value (str (:group-id cache-id) "/" (:artifact-id cache-id))}]
-     [:input.pa2.mr2.br2.ba.outline-0.blue {:type "hidden" :id "version" :name "version" :value (:version cache-id)}]
-     [:input.f7.white.hover-near-white.outline-0.bn.bg-white {:type "submit" :value "rebuild"}]]
-    (if scm-url
-      [:a.link.dim.gray.f6.tr
-       {:href scm-url}
-       [:img.v-mid.mr2 {:src "https://icon.now.sh/github"}]
-       [:span.dib (util/gh-coordinate scm-url)]]
-      [:a.f6.link.blue {:href (util/github-url :userguide/scm-faq)} "SCM info missing"])]])
+(defn top-bar
+  ([]
```

Comment:
I added this function overload. Not sure if there is a more idiomatic way.

Reply:
I think an alternative approach could have been to create a separate function `top-bar-generic` (since the difference is that it's not specific to a particular project). 

This would also keep the diff smaller making it potentially easier to follow for reviewers.

Reply:
Sure, I can do that! Should I share the markup or is the duplication alright?